src/main/io/serial_cli.c

```diff
@@ -500,6 +500,10 @@ static const char * const lookupTableLowpassType[] = {
     "NORMAL", "HIGH"
 };
 
+static const char * const lookupTableLedAux[] = {
+    "OFF", "1","2","3","4"
```

Comment:
I used it to offer option "OFF" in cli instead of "0".  Maybe it will be usefull, if someone wants to add the option to configurator? I don´t know if its possible to make the lookupTable dynamic depending on MAX_AUX_CHANNELS?